x-pack/plugins/maps/public/classes/layers/wizards/layer_wizard_registry.ts

```diff
@@ -90,3 +91,7 @@ export async function getLayerWizards(): Promise<LayerWizardWithMeta[]> {
       return wizard1.order - wizard2.order;
     });
 }
+
+export function getWizardById(autoOpenLayerWizardId: string) {
```

Comment:
Looks like global search and replace error, `autoOpenLayerWizardId` should be `wizardId` here